source/fundamentals/indexes.txt

```diff
@@ -117,20 +126,19 @@ See the MongoDB server manual section on :manual:`Compound indexes </core/index-
 Multikey Indexes (Indexes on Array Fields)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Multikey indexes** are indexes that improve performance for queries that specify ascending or descending indexes on
-fields that contain an array value. You can define a multikey index using the same syntax as a single field or compound
-index.
+**Multikey indexes** are indexes that improve performance for queries that specify an index on a field that contains an
```

Comment:
Huge improvement here -- this gets to the point with a lot fewer words. Would this be even clearer if we removed "that specify an index", since queries do not have to explicitly specify an index to use it? Or do you think that's cutting too many words?

Reply:
I've reworded this slightly, good catch. The emphasis should be on the field queried, which I think it is now.